src/TaskRunner/Commands/DrupalCommands.php

```diff
@@ -620,8 +620,8 @@ protected function getToolkitSettingsBlock()
 \$settings['file_temp_path'] = getenv('DRUPAL_FILE_TEMP_PATH') !== FALSE ? getenv('DRUPAL_FILE_TEMP_PATH') : '/tmp';
 
 // Reverse proxy.
-if (intval(getenv('DRUPAL_REVERSE_PROXY_ENABLE')) === 1) {
-  \$settings['reverse_proxy'] = (bool) getenv('DRUPAL_REVERSE_PROXY_ENABLE');
+if (filter_var(getenv('DRUPAL_REVERSE_PROXY_ENABLE'), FILTER_VALIDATE_BOOLEAN)) {
+  \$settings['reverse_proxy'] = getenv('DRUPAL_REVERSE_PROXY_ENABLE');
```

Comment:
Shouldn't it be like this:  $settings['reverse_proxy'] = true; ? Usage of FILTER_VALIDATE_BOOLEAN can handle multiple truthy values like "1", "true", "on" and "yes" and assigns the raw value of getenv('DRUPAL_REVERSE_PROXY_ENABLE') to $settings['reverse_proxy'], which could be a string ("true", "1", etc.).

Reply:
Hi @ozyrys ,
The problem is the different infrastructures, in one side it should be `True` and another should be `1`.
To make this more flexible, I decided to let it just receive the value defined by each project, but using the `filter_var()` just to ensure that is set and is any kind of positive.
Let me know if it make sense.